src/config/types.ts

```diff
@@ -223,6 +223,10 @@ export class BoundaryLayerProps extends CommonLayerProps {
   adminLevelNames: string[]; // Ordered (Admin1, Admin2, ...)
   adminLevelLocalNames: string[]; // Same as above, local to country
   styles: LayerStyleProps; // Mapbox line and fill properties.
+  @optional
+  primary?: boolean; // Flag for general default layer a replacement for `admin_boundaries`
```

Comment:
Do we need this new flag? Why not just keep `admin_boundaries` or maybe change it to `default_admin_boundaries` if you prefer?

Reply:
This way we don't need to add any new flag